src/js/themes/hpe.js

```diff
@@ -106,31 +106,28 @@ export const hpe = deepFreeze({
       face: `
         @font-face {
           font-family: "Metric";
-          src: url("https://d3hq6blov2iije.cloudfront.net/fonts/HPEXS-Metric-Fonts/MetricHPEXS-Regular.woff2") format('woff2'),
-               url("https://d3hq6blov2iije.cloudfront.net/fonts/HPEXS-Metric-Fonts/MetricHPEXS-Regular.woff") format('woff');
+          src: url("https://www.hpe.com/h41225/hfws-static/fonts/metric-hpe/MetricHPE-Web-Regular.woff") format('woff');
```

Comment:
We should be providing the woff2 versions for all of these fonts. As far as I can tell, we have them all in the email, but the file naming convention is inconsistent.

Reply:
From the urls, I interpreted that MetricHPE-Web was the path we wanted to use, but just sent an email to Artur to follow up about any differences between MetricHPE and Metric from the urls.